src/PhpPact/Consumer/Listener/PactTestListener.php

```diff
@@ -58,20 +58,20 @@ public function __construct(array $testSuiteNames)
      *
      * @throws \Exception
      */
-    public function startTestSuite(TestSuite $suite)
+    public function startTestSuite(TestSuite $suite): void
```

Comment:
This library should be compatible with PHP 7.0, but void return was introduced only in PHP 7.1 (see http://php.net/manual/en/migration71.new-features.php).